paimon-core/src/test/java/org/apache/paimon/table/AppendOnlyFileStoreTableTest.java

```diff
@@ -241,6 +265,31 @@ public void testBatchSplitOrderByPartition() throws Exception {
         assertThat(partitions).containsExactly(1, 2, 3);
     }
 
+    @Test
+    public void testBranchStreamingReadWrite() throws Exception {
+        FileStoreTable table = createFileStoreTable();
+        generateBranch(table);
+        writeBranchData(table);
+
+        List<Split> splits =
+                toSplits(
+                        table.newSnapshotReader(BRANCH_NAME)
+                                .withMode(ScanMode.DELTA)
+                                .read()
+                                .dataSplits());
+        TableRead read = table.newRead();
+        System.out.println(getResult(read, splits, binaryRow(1), 0, STREAMING_ROW_TO_STRING));
```

Comment:
We should not use `System.out.pringln` in tests

Reply:
Thanks for your reminder. Done. @FangYongs